src/components/shared/nav-bar/nav-bar.module.scss

```diff
@@ -0,0 +1,68 @@
+@use '@variables' as var;
+@use '@mixins' as mixins;
+
+.navigation {
+  position: sticky;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 auto;
+  width: 100%;
+  max-width: 1131px;
+  height: 120px;
+  z-index: 100;
+  @include mixins.padding;
+  background-color: var.$white;
+}
+
+.nextLink {
+  color: var.$black;
+
+  &.activeLink {
+    font-weight: 700;
+  }
+}
+
+.magnifyingGlass {
```

Comment:
The magnifying glass isnt in this component anymore. Can this be moved / removed?